recipe/meta.yaml

```diff
@@ -1,12 +1,13 @@
 
 package:
   name: usgscsm
-  version: "0.1.0"
+  version: {{ environ.get("GIT_DESCRIBE_TAG", "") }}
 
 source:
   git_url: https://github.com/USGS-Astrogeology/CSM-CameraModel.git
 build:
-  string: dev
+  number: {{ environ.get("GIT_DESCRIBE_NUMBER", 0) }}
+  string: {{ environ.get("GIT_DESCRIBE_HASH", "") }}
```

Comment:
We don't need both the number of commits since tag and the current commit. We can just use the current HEAD commit.